flutter/ios/sentry_flutter/Sources/sentry_flutter/SentryFlutterPlugin.swift

```diff
@@ -11,7 +17,7 @@ import CoreVideo
 // swiftlint:disable file_length function_body_length
 
 // swiftlint:disable:next type_body_length
-public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
+public class SentryFlutterPlugin: NSObject, FlutterPlugin {
```

Comment:
is this needed or just an improvement from your side?

Reply:
not against this btw, just curious

Reply:
Needed as in SPM, we can't import mixed language. So i removed the redundant obj-c implementation that just delegated to the swift implementation and use just this file through interop.